packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -361,7 +361,7 @@ export const makeZCFZygote = async (
       privateArgs = undefined,
     ) => {
       zoeInstanceAdmin = instanceAdminFromZoe;
-      initSeatMgrAndMintFactory();
+      void initSeatMgrAndMintFactory();
```

Comment:
this should be awaited. if it fails the contract should not restart

Reply:
Done